controls/cis_rhel8.yml

```diff
@@ -971,15 +971,14 @@ controls:
       - service_xinetd_disabled
 
   - id: 2.2.20
-    title:  Ensure rsync is not installed or the rsyncd service is masked (Automated)
+    title: Ensure X window server services are not in use (Automated)
     levels:
-      - l1_server
-      - l1_workstation
+      - l2_server
     status: automated
+    notes: >-
+      The rule also configures correct run level to prevent unbootable system.
```

Comment:
The runlevel is configured by `xwindows_runlevel_target`, but it is not selected here.